plugins/modules/notification/cisco_webex.py

```diff
@@ -10,99 +10,102 @@
-    required: False
+       - Specifies how you would like the msg formatted.
     default: text
     choices: ['text', 'markdown']
+    type: str
 
   personal_token:
     description:
-      - Your personal access token required to validate the Spark API.
-    required: true
-    aliases: ['token']
+      - Your personal access token required to validate the Webex Teams API.
+    required: yes
+    type: str
 
-  message:
+  msg:
     description:
       - The message you would like to send.
-    required: True
+    required: yes
+    type: str
 '''
 
 EXAMPLES = """
 # Note: The following examples assume a variable file has been imported
 # that contains the appropriate information.
 
-- name: Cisco Spark - Markdown Message to a Room
-  cisco_spark:
+- name: Cisco Webex Teams - Markdown Message to a Room
+  cisco_webex:
     recipient_type: roomId
     recipient_id: "{{ room_id }}"
-    message_type: markdown
+    msg_type: markdown
     personal_token: "{{ token }}"
-    message: "**Cisco Spark Ansible Module - Room Message in Markdown**"
+    msg: "**Cisco Webex Teams Ansible Module - Room Message in Markdown**"
 
-- name: Cisco Spark - Text Message to a Room
-  cisco_spark:
+- name: Cisco Webex Teams - Text Message to a Room
+  cisco_webex:
     recipient_type: roomId
     recipient_id: "{{ room_id }}"
-    message_type: text
+    msg_type: text
     personal_token: "{{ token }}"
-    message: "Cisco Spark Ansible Module - Room Message in Text"
+    msg: "Cisco Webex Teams Ansible Module - Room Message in Text"
 
-- name: Cisco Spark - Text Message by an Individuals ID
-  cisco_spark:
+- name: Cisco Webex Teams - Text Message by an Individuals ID
+  cisco_webex:
     recipient_type: toPersonId
     recipient_id: "{{ person_id}}"
-    message_type: text
+    msg_type: text
     personal_token: "{{ token }}"
-    message: "Cisco Spark Ansible Module - Text Message to Individual by ID"
+    msg: "Cisco Webex Teams Ansible Module - Text Message to Individual by ID"
 
-- name: Cisco Spark - Text Message by an Individuals E-Mail Address
-  cisco_spark:
+- name: Cisco Webex Teams - Text Message by an Individuals E-Mail Address
+  cisco_webex:
     recipient_type: toPersonEmail
     recipient_id: "{{ person_email }}"
-    message_type: text
+    msg_type: text
     personal_token: "{{ token }}"
-    message: "Cisco Spark Ansible Module - Text Message to Individual by E-Mail"
+    msg: "Cisco Webex Teams Ansible Module - Text Message to Individual by E-Mail"
 
 """
 
 RETURN = """
 status_code:
   description:
-    - The Response Code returned by the Spark API.
-    - Full Response Code explanations can be found at U(https://developer.ciscospark.com/endpoint-messages-post.html).
+    - The Response Code returned by the Webex Teams API.
+    - Full Response Code explanations can be found at U(https://developer.webex.com/docs/api/basics).
   returned: always
   type: int
   sample: 200
 
-message:
+msg:
```

Comment:
You are removing an existing return value. The old one needs to be deprecated (and not removed) and returned until the deprecation period is over.

Reply:
there was a collision with the name of the message and therefore the module stopped working, hence the change from message to msg.

Reply:
That's for module options, not for return values. Please keep the old name of the return value.